pysmt/cmd/installers/base.py

```diff
@@ -0,0 +1,205 @@
+import os
+import platform
+import sys
+import shutil
+
+from six.moves import xrange
+from six.moves.urllib import request as urllib2
+from six.moves.urllib.error import HTTPError
+
+from pysmt.cmd.installers.utils import unzip, untar
```

Comment:
I think it would be better to move those functions within this class. Base should provide all the functionalities that you need.